### DISCUSSION TITLE
Pull Requests

### DISCUSSION THREAD

**Discussion by Ayeshamalikdev** [community/community#82239] [Pull Requests]
### Select Topic Area

Question

### Body

Hey there,
How can we pull request?

**Comment by MastooraTurkmen** [comment]
Hey,
when you commit your repo you can click to pull a request.

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
